crates/astria-sequencer/justfile

```
@@ -28,18 +28,18 @@ run-testnet:
 
 [linux]
 stop-testnet:
-  pkill cometbft && pkill astria-sequencer
+  pkill cometbft && pkill -f astria-sequencer
```

The `-f` causes `pkill` to search the full command line rather than just the process name, which ends up killing not only any running `astria-sequencer` but also the shell which `just` spawns to run the command.  The full command which `just` executes is
```sh
sh -cu pkill cometbft && pkill -f astria-sequencer
```
So `pkill` kills that shell, causing the `just` execution to fail.

We can specify `pkill -f ^astria-sequencer` to ensure it's matching only on the start of the command.

Also, we can probably ignore the results of the `pkill` calls, meaning we can then make `stop-testnet` a dependency of `clean-testnet` again?

```suggestion
  pkill cometbft || true
  pkill -f ^astria-sequencer || true
```

If the `[macos]` variant of `pkill` supports `-f` in the same way, we could maybe deduplicate these sections?

---

good point, i was wondering why the justfile  was always failing lmao. `pkill -f ^astria-sequencer` didn't end up killing the sequencer processes for me as it wasn't an exact match. i changed this to be `pkill -f [a]stria-sequencer` which still regex-matches processes with `astria-sequencer` but not the shell as it no longer contains `astria-sequencer` :D